vllm_spyre/platform.py

```diff
@@ -19,13 +19,13 @@
 
 class SpyrePlatform(Platform):
     _enum = PlatformEnum.OOT
-    device_name: str = "spyre"
+    device_name: str = "cpu"
```

Comment:
Oh let's also put a comment about this hack here too for future readers

Reply:
Done!